### PR TITLE
Feature dashboard

### PR DESCRIPTION
**Pull request by aneesps**
@nishintn 
Completed the layout of page.

### CODE REVIEW

File: views/dailystatus.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Space - Daily Status</title>
+	<script type="text/javascript" src="../assets/js/lib/angular.min.js"></script>
```
Comment (mad-nishin) [inline views/dailystatus.html]: Why did you refer anular.js here? This should be referred in main index file rt?

**Comment by mad-nishin** [comment]
@aneesps ,
Also where do you initialize the angular app? ng-app?

**Comment by hawariotm** [comment]
yes it should be refered at index. but now i am trying to implement the
functionality of the page. we didnt implemented it as partials. now every
page is single entity. we will convert into partials later after
implementing functionality.

i initilazed ng-app in a separate div . not in <html> tag.

**Comment by aneesps** [comment]
@nishintn ,
I was trying to achieve the functionality of the page. So I designed it as a single entity,hence I needed to refer angular.js in the page, 'dailystatus.html'.
I have initialized ng-app on the first div element.

**Comment by mad-nishin** [comment]
@hawariotm  @aneesps ,
I dont recommend rework. Implement this as partials.

**Comment by aneesps** [comment]
@nishintn ,
I will implement this as partials and will send you a new pull request.

**Comment by aneesps** [comment]
@nishintn 
hai,
I have added the basic layout of the pages including the navigation menu and have removed the 'dailystatus.html' from this branch. Please merge this branch to master.

**Comment by aneesps** [comment]
@nishintn ,
I have fixed the mistakes in this branch. I request you to merge this branch to master so that everybody can pull the master and use this layout.

**Pull request merged**